backend/main.py

```diff
@@ -74,13 +75,84 @@ def SignIn():
             #One-way hash inputted password to see if it matches db records. If no match, raises exception
             verify_result = salted_hasher.verify(db_hashed_passwd, SignIn_data.get("password"))
             #If passed without exceptions:
-            response_object["username"] = db_username
-            response_object["email"] = db_email
-            response_object["message"] = "User account found and logged in"
+            response["username"] = db_username
+            response["email"] = db_email
+            response["message"] = "User account found and logged in"
         except:
             cursor.close()  
-            response_object["message"] = "User account not found"
-    return jsonify(response_object)
+            response["message"] = "User account not found"
+    return jsonify(response)
+
+@app.route("/PredPrey/AlterPresets", methods=["POST"])
+def add_PredPrey_preset():
+    response = {"server status": "success"}
+    preset = request.get_json() #Retrieve payload from client
+    if request.method == "POST": #Adding a preset
```

Comment:
why do you need if `request.method==post`? if the route is defined as a post request that should be the only possible way you can use the route

Reply:
I though that the default method for Flask was GET, so I have to specify POST. Might be wrong though

Reply:
The method(s) should be whatever you define in the decorator of the route

Reply:
Ok sure, think it's only default GET if you d
on't define any